Solutions/TenableIO/Data Connectors/host.json

```diff
@@ -1,6 +1,10 @@
 {
   "version": "2.0",
   "logging": {
+	"logLevel": {
```

Comment:
Spacing looks like it should be fixed here

Reply:
Fixed, package (zip file) does not have this spacing issue.